shopelectro/management/commands/excel.py

```diff
@@ -112,7 +112,7 @@ def collapse_row(row):
         def write_product_rows():
             """Write products lines."""
             sheet = self.sheet
-            products = Product.objects.filter(category=category)
+            products = Product.objects.filter(category=category, page__is_active=True)
```

Comment:
let's add custom ModelManager instead:

```
class ProductManager(models.Manager):
    def get_queryset(self):
        return super(DahlBookManager, self).get_queryset().filter(author='Roald Dahl')

class Product:
    active = ProductManager()

# in client code
Product.active.filter(category=category)
```

Took example here: https://docs.djangoproject.com/en/1.11/topics/db/managers/#modifying-a-manager-s-initial-queryset

Reply:
@duker33 I am going to create todo for it and it seems that this code will not work, but we may discuss it in the related issue